app/App.jsx

```diff
@@ -3,17 +3,28 @@ import styles from "./App.module.css";
 import Card from "./src/ui/components/Card.jsx";
 import Container from "./src/ui/components/Container.jsx";
 import { useTheme } from "./src/ui/hooks/useTheme.js";
+import Switch from "./src/ui/components/Switch.jsx";
 import clsx from "clsx";
 
 const App = () => {
-  const { restoreTheme, setTheme } = useTheme();
+  const { restoreTheme, setTheme, getTheme } = useTheme();
 
+  const handleCheckboxChange = () => {
+    setTheme(getTheme() === "light" ? "dark" : "light");
+  };
+  const booleanValue = getTheme() === "light" ? true : false;
```

Comment:
name it properly please

read this article to understand how to name boolean variables so it can tell you what this variable about just by simply looking at its name:
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig